code/src/Components/NavButton.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+const NavButton = (props) => {
+    
+    return (
+        <button 
+            className="nav-button"
+            onClick={() => props.navigate(props.direction)}
+        >{props.direction === -1 ? "<" : ">"}</button>
+    )                           //this might fail cuz strings??
```

Comment:
Love the ternary operator!!! I could not come up with one for the button's direction. Mine only go one way